internal/godojs/godojs.go

```diff
@@ -738,6 +888,85 @@ func TagToVM(vm *otto.Otto, g *godo.Tag) otto.Value {
 	})
 }
 
+func LoadBalancerToVM(vm *otto.Otto, g *godo.LoadBalancer) otto.Value {
+	if g == nil {
+		return otto.NullValue()
+	}
+
+	return ottoutil.ToPkg(vm, map[string]interface{}{
+		"id":                     g.ID,
+		"name":                   g.Name,
+		"ip":                     g.IP,
+		"algorithm":              g.Algorithm,
+		"status":                 g.Status,
+		"created_at":             g.Created,
+		"forwarding_rules":       ForwardingRulesToVM(vm, g.ForwardingRules),
+		"health_check":           HealthCheckToVM(vm, g.HealthCheck),
+		"sticky_sessions":        StickySessionsToVM(vm, g.StickySessions),
+		"region":                 RegionToVM(vm, g.Region),
+		"tag":                    g.Tag,
+		"droplet_ids":            intsToInt64s(g.DropletIDs),
+		"redirect_http_to_https": g.RedirectHttpToHttps,
+	})
+}
+
+func ForwardingRulesToVM(vm *otto.Otto, g []godo.ForwardingRule) otto.Value {
+	if g == nil {
+		return otto.NullValue()
+	}
+
+	var rules = make([]map[string]interface{}, 0)
```

Comment:
`rules := make([]map[string]interface{}, 0)` or `var rules []map[string]interface{}` but not both :P